packages/navuser/src/css/index.js

```diff
@@ -0,0 +1,45 @@
+import {
+  colorsBackgroundUtility,
+  colorsTextIcon,
+  layout,
+  type
+} from '@pluralsight/ps-design-system-core'
+import { transparentize } from '@pluralsight/ps-design-system-util'
+
+export default {
+  '.psds-navuser': {
+    display: 'inline-flex',
+    alignItems: 'center',
+    padding: `${layout.spacingXXSmall} ${layout.spacingXSmall}`,
+    textDecoration: 'none',
+    border: 'none',
+    background: 'none',
+    borderRadius: '2px'
+  },
+  '.psds-navuser--clickable': {
+    '&:hover, &:focus': {
+      background: transparentize(0.85, colorsBackgroundUtility.base)
+    },
+    '&:focus': {
+      outline: 'none'
+    },
+    '&:active': {
+      background: transparentize(0.75, colorsBackgroundUtility.base)
+    }
+  },
+  '.psds-navuser__plan-name': {
```

Comment:
wondering about an upperlimit on text. should we cap/truncate

cc @sweeting

Reply:
We are going to do a text overflow on a maxwidth on the container.  I need to add that.